verilog/tools/ls/vscode/src/download-ls.ts

```diff
@@ -0,0 +1,108 @@
+import { platform, arch } from "os";
+import * as fs from "fs";
+import * as path from "path";
+import { IncomingMessage } from "http";
+import { https as httpsFr } from "follow-redirects";
+import { execSync } from "child_process";
+import * as decompress from "decompress";
+const decompressTargz = require("decompress-targz");
+const TAG = require("../package.json").repository.tag;
+
+
+function checkIfBinaryExists(binaryPath: string) {
+  let whichCommand: string, binaryExists: boolean;
+  if (platform() == "win32") whichCommand = "where";
+  else whichCommand = "which";
+
+  binaryExists = true;
+  try {
+    execSync(`${whichCommand} ${binaryPath}`, { windowsHide: true });
+  } catch {
+    binaryExists = false;
+  }
+
+  return binaryExists;
+}
+
+export async function checkAndDownloadBinaries(
+  binaryPath: string
+): Promise<string> {
+  if (platform() === "darwin") {
+    // There is no static binaries for MacOS -- aborting
```

Comment:
Maybe file an issue that we don't forget to supply one if possible.
There is also https://github.com/chipsalliance/homebrew-verible maybe we can make that work with regular updates.

Reply:
I added information that Verible can be installed with `brew`.

I also created issue #1897